charts/internal/calico/values.yaml

```diff
@@ -24,4 +24,6 @@ images:
   calico-kube-controllers: "image-repository:image-tag"
   calico-podtodaemon-flex: "image-repository:image-tag"
   calico-cpva: "image-repository:image-tag"
-  calico-cpa: "image-repository:image-tag"
\ No newline at end of file
+  calico-cpa: "image-repository:image-tag"
+
+nodeSelector: {}
```

Comment:
If you decide to implement the above suggestion, you have to comment this line.
```suggestion
# nodeSelector: {}
```

Reply:
Done